.github/bors.toml

```diff
@@ -0,0 +1,3 @@
+block_labels = ["I-needs-decision", "S-blocked"]
```

Comment:
I left this as inspiration, do we want labels that prevent PRs from being merged?

Reply:
I don't think we're *that* big yet: most of the time PRs get merged pretty quick. We only have the current situation due to the issue with Travis. When we do hit the point where such things are needed it can always be added later, I think.